guardian/manage-frontend
# Integrate update to discount-api

## What does this change?

Update the response from discount-api as of [this PR](https://github.com/guardian/support-service-lambdas/pull/2133). No longer calculate the discount rate plan or number of months it's applicable for, but let the api tell us. The api also returns a 400 instead of a boolean when the subscription is ineligible for a discount.



## How to test

With a digisub, go to `/digital?withFeature=digisubSave` and cancel - should see discount including how many months it's valid for



## How can we measure success?



## Have we considered potential risks?



## Images



## Accessibility
